### PR TITLE
Update readme + scripts

### PR DESCRIPTION
**Pull request by nenonja**
## Description :sparkles:
Readme.md was out of date and doesnt match to current scripts. Now updated

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,29 +1,48 @@
...
-1. Copy the contents of `.env.benefit-*.example` to `.env.benefit-*` and modify them if needed.
+1. Run `yarn` to install necessary packages
+
+2. Copy the contents of `.env.benefit-*.example` to `.env.benefit-*` and modify them if needed.
 
-2. Run `docker-compose -f docker-compose.benefit.yml up`
+3. Run `yarn benefit` or, if you want to rebuild, then `yarn benefit --build`
 
 The Applicant Frontend is now running at [localhost:3000](https://localhost:3000)
 The Handler Frontend is now running at [localhost:3100](https://localhost:3100)
```
Comment (karisal-anders) [inline README.md:48]: Now the two last rows are shown actually on a single row when viewing the README.md through a markdown viewer 
 and not on different rows.

To fix this they could e.g. be put under the "3." section:
`
3. Run `yarn youth` or, if you want to rebuild, then `yarn youth --build`
   - The Frontend is now running at [localhost:3100](https://localhost:3000)
   - The backend is now running at [localhost:8000](https://localhost:8000)
`

File: README.md [hunk 2]
```diff
@@ -1,29 +1,48 @@
...
-1. Copy the contents of `.env.kesaseteli.example` to `.env.kesaseteli` and modify it if needed.
+1. Run `yarn` to install necessary packages
 
-2. Run `yarn kesaseteli` or, if you want to rebuild, then `yarn kesaseteli --build`
+2. Copy the contents of `.env.kesaseteli.example` to `.env.kesaseteli` and modify it if needed.
+
+3. Run `yarn employer` or, if you want to rebuild, then `yarn employer --build`
 
 The Frontend is now running at [localhost:3000](https://localhost:3000)
 The backend is now running at [localhost:8000](https://localhost:8000)
```
Comment (karisal-anders) [inline README.md:16]: Similar thing as in https://github.com/City-of-Helsinki/yjdh/pull/542/commits/6802d328a8ed55fa34b7f53ad8303d1d0213a5c7#r767705181

File: README.md [hunk 3]
```diff
@@ -1,29 +1,48 @@
...
 
 3. If services fail to get up, `yarn clean` might help.
```
Comment (karisal-anders) [inline README.md:18]: Now the "The Frontend" and "The backend" rows are shown actually on a single row when viewing the README.md through a markdown viewer and not on different rows. The number of the last entry in the list should be updated too.

To fix this they could e.g. be put under the "3." section:
```
3. Run `yarn employer` or, if you want to rebuild, then `yarn employer --build`
   - The Frontend is now running at [localhost:3000](https://localhost:3000)
   - The backend is now running at [localhost:8000](https://localhost:8000)
4. If services fail to get up, `yarn clean` might help.
```

File: README.md [hunk 4]
```diff
@@ -1,29 +1,48 @@
...
+### kesaseteli-youth Development with Docker
+
+1. Run `yarn` to install necessary packages
+
+2. Copy the contents of `.env.kesaseteli.example` to `.env.kesaseteli` and modify it if needed.
+
+3. Run `yarn youth` or, if you want to rebuild, then `yarn youth --build`
+
+The Frontend is now running at [localhost:3100](https://localhost:3000)
+The backend is now running at [localhost:8000](https://localhost:8000)
```
Comment (karisal-anders) [inline README.md:33]: Now the two last rows are shown actually on a single row when viewing the README.md through a markdown viewer and not on different rows.

To fix this they could e.g. be put under the "3." section:
```
3. Run `yarn youth` or, if you want to rebuild, then `yarn youth --build`
   - The Frontend is now running at [localhost:3100](https://localhost:3000)
   - The backend is now running at [localhost:8000](https://localhost:8000)
```

Comment (nenonja) [reply README.md]: fixed

File: README.md (hunk 3 above)
Comment (nenonja) [reply README.md]: fixed

File: README.md (hunk 1 above)
Comment (nenonja) [reply README.md]: fixed

**Review by karisal-anders — APPROVED** [review]
Looks good to me! 👌

**Pull request merged**